docs/contributing.md

```diff
@@ -0,0 +1,48 @@
+# SSS Contributor Guideline
+
```

Comment:
Add a short intro: "These are recommendations when contributing to the SSS ... They consider contributions to both actual content (mostly Markdown) and source code (challenges and activities) done via Git. Please consider them when making contributions ...."